src/source/shift.rs

```diff
@@ -9,7 +9,9 @@ use std::path::*;
 use std::process::*;
 use std::sync::Mutex;
 
+use binjs_es6::ast::Script as ScriptAST;
```

Comment:
Not a big fan of `ScriptAST` as a name. Maybe `ES6ScriptRoot`?

Reply:
Maybe... Could as well just use `AST` or just import `binjs_es6::ast` and use it as `ast::Script` everywhere. Idk.

Reply:
Or maybe I should rename local `Script` to something like `Bindings`?

Reply:
Yeah, let's call it `AST` for the moment. If we ever extend this to other languages (e.g. JSON), we'll have time to change name.

Reply:
Done.